.travis.yml

```diff
@@ -13,28 +13,30 @@ matrix:
       env: OPENPGP_NODE_JS='8' OPENPGPJSTEST='unit'
     - node_js: "10"
       env: OPENPGP_NODE_JS='10' OPENPGPJSTEST='unit'
-    - node_js: "9"
+    - node_js: "node"
+      env: OPENPGP_NODE_JS='10' OPENPGPJSTEST='unit'
+    - node_js: "10"
       env: BROWSER='Firefox' VERSION='61' PLATFORM='macOS 10.13' OPENPGPJSTEST='saucelabs'
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Chrome' VERSION='49' PLATFORM='macOS 10.13' OPENPGPJSTEST='saucelabs' COMPAT=1
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Chrome' VERSION='68' PLATFORM='macOS 10.13' OPENPGPJSTEST='saucelabs'
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Internet Explorer' VERSION='11.103' PLATFORM='Windows 10' OPENPGPJSTEST='saucelabs' COMPAT=1
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='MicrosoftEdge' VERSION='17.17134' PLATFORM='Windows 10' OPENPGPJSTEST='saucelabs'
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Safari' VERSION='10' PLATFORM='macOS 10.12' OPENPGPJSTEST='saucelabs' COMPAT=1
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Safari' VERSION='11' PLATFORM='macOS 10.13' OPENPGPJSTEST='saucelabs'
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='Android' VERSION='6.0' OPENPGPJSTEST='saucelabs'
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='iPad' VERSION='10.0' OPENPGPJSTEST='saucelabs' COMPAT=1
-    - node_js: "9"
+    - node_js: "10"
       env: BROWSER='iPad' VERSION='11.0' OPENPGPJSTEST='saucelabs'
   allow_failures:
-    - node_js: "9"
+    - node_js: "10"
```

Comment:
This causes the Node.js 10 test to be allowed to fail, in addition to the browser tests. Since it's a simple string match, you could change it (and the `node_js` versions of the browser tests above) to "10.0" or "11". But since we're testing browsers here, and not the node version we're running them on, leaving it at 9 should also be fine.

Or you could just remove the Node.js 10 test, since I think it's a bit unlikely that OpenPGP.js would work in Node 8 and 11 but be broken in 10. But I'll leave it up to you.

Reply:
> leaving it at 9 should also be fine.

Which is not needed imho as Node.js 9 reached its EOL and should not be used in production.

Reply:
I'll probably remove the `allow_failures` entries.

Reply:
The `allow_failures` entry is still needed here for the browser tests, which often fail for no particular reason. The reason it stated `node_js: "9"` is not because Node 9 is allowed to fail, but because all the browser tests ran on Node 9.

Reply:
> but because all the browser tests ran on Node 9

Well, most libraries are not tested against Node.js 9 (anymore) and it will not receive any bugfixes and security fixes. If so, we can filter by `OPENPGPJSTEST` or similar.

Reply:
Travis doesn't match by individual env variables.

Reply:
Mh, that might be a bit more complicated. See https://github.com/travis-ci/travis-ci/issues/6661

Reply:
But isn't Node.js also used for building the library when running the browser tests? So we can not just set the version to something different and think that this is enough. I understand that this is currently done to split tests and allow failing Saucelabs test. But should they be really allowed to fail?

Reply:
Yes, but building the library is also done in the Node tests. Besides, that doesn't really differ at all between different versions of Node.